apollo-backend/src/main/java/io/logz/apollo/notifications/senders/MarkersSender.java

```diff
@@ -0,0 +1,100 @@
+package io.logz.apollo.notifications.senders;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.collect.ImmutableList;
+import io.logz.apollo.common.HttpStatus;
+import io.logz.apollo.models.DeploymentMarker;
+import io.logz.apollo.models.DeploymentMarkers;
+import io.logz.apollo.notifications.NotificationTemplateMetadata;
+import okhttp3.MediaType;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.RequestBody;
+import org.mariadb.jdbc.internal.logging.Logger;
+import org.mariadb.jdbc.internal.logging.LoggerFactory;
+
+import java.io.IOException;
+import java.util.HashMap;
+
+public class MarkersSender implements NotificationSender {
+    private static final Logger logger = LoggerFactory.getLogger(MarkersSender.class);
+    private final String X_API_TOKEN_HEADER = "X-API-TOKEN";
+    private final String apiToken;
+    private final String createMarkersEndpointUrl;
+
+    public MarkersSender(String notificationJsonConfiguration) throws IOException {
+        ObjectMapper mapper = new ObjectMapper();
+        DeploymentMarkersConfiguration markersConfiguration = mapper.readValue(notificationJsonConfiguration, DeploymentMarkersConfiguration.class);
+        this.apiToken = markersConfiguration.getApiToken();
+        this.createMarkersEndpointUrl = markersConfiguration.getCreateMarkersEndpointUrl();
+    }
+
+    @Override
+    public boolean send(NotificationTemplateMetadata notificationTemplateMetadata) {
+        OkHttpClient httpClient = new OkHttpClient();
+        Request.Builder builder = new Request.Builder().url(this.createMarkersEndpointUrl)
+                .header(X_API_TOKEN_HEADER, this.apiToken);
+
+        DeploymentMarkers deploymentMarkers = getDeploymentMarkers(notificationTemplateMetadata);
+        executeMarkersRequest(httpClient, builder, deploymentMarkers);
+        return true;
```

Comment:
oh..
you return true no matter what,
so processQueue in ApolloNotifications will not retry your send function
is it intentional?

Reply:
I'm not that familiar with Apollo's retry mechanism (times etc), I can `return false` if the http request has failed. wdyt?

Reply:
you can return false if http request to markers enpdoint failed in `io.logz.apollo.notifications.senders.MarkersSender#executeMarkersRequest`. 
Then code in `ApolloNotifications#processQueue` will try it 2 more times.

Reply:
I'll do that 👍